test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/Transform.test.cpp

```diff
@@ -140,13 +268,38 @@ TEST(OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_Transform, Validation)
             for (const auto& referenceRow : referenceData)
             {
                 const Instant instant = Instant::DateTime(DateTime::Parse(referenceRow[0].accessString()), Scale::TAI);
+
                 Shared<const Frame> frame1 = Frame::GCRF();
                 Shared<const Frame> frame2 = Frame::GCRF();
 
                 if (frame2Str == "TOD")
                 {
                     frame2 = Frame::TOD(instant, iau::TheoryFromString(otherStr));
                 }
+                else if (frame2Str == "TEME")
```

Comment:
Do we have a `String2Frame` function? 🤔 (just curious)

Reply:
There is a `Frame::WithName('AName')` constructor. It uses a map in the manager but I can't see how it's populated, so I'll try it out and see what happens

Reply:
I don't think it can be used. I think it might even be slightly 'broken'. See https://loftorbital.atlassian.net/browse/CPT-8882